src/utils/switch-site.ts

```diff
@@ -0,0 +1,38 @@
+import { $theme } from "../store/theme"
+import { ThemeKey } from "../types/theme"
+import { getCurrentTheme } from "./current-theme"
+import { login } from "./login"
+import { logout } from "./logout"
+import { queryClient } from "./query-client"
+
+const SITE_TO_USER_MAP: Record<ThemeKey, string> = {
+  stitch: "rene@example.com",
+  luminara: "cecilia@example.com",
+  pug: "emily@example.com",
+}
+
+export async function loginToSite(_site: ThemeKey | null) {
```

Comment:
nit: I'd just rename everything relating to `theme` to `site` instead. I feels weird to use `ThemeKey` type for `site` variable, but it feels ok to use `Site` (type) for `theme` variable don't you think?